tests/functional.rs

```diff
@@ -590,7 +590,7 @@ async fn functional() {
     info("##################################################################");
     info("");
     {
-        refresh_all().await;
+        cas_all_sync_parents().await;
```

Comment:
Ditto: was refresh, now sync, am confused about the terminology and wondering if this is correct?

Reply:
using 'refresh' again